cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/BillingBudgetFilterDTO.java

```diff
@@ -16,6 +16,9 @@ public class BillingBudgetFilterDTO implements DTO
     @Valid
     private BillingBudgetPeriodDTO period = new BillingBudgetPeriodDTO( CalendarPeriod.MONTH );
 
+    @ApiModelProperty( "Filter by single subscription" )
+    private UrlEntityDTO subscription;
```

Comment:
forse integer `subscriptionId`?

nella post budge estimate si andava per id: https://docs.cloudesire.com/redoc/billing.html#operation/budgetEstimateRequest

Reply:
non ricordo la ratio per la quale si sia a volte usato l'urlEntity `subscription/id` e altre il semplice `id`

direi che se il riferimento è salvato in un'entità (tipo qui) va usato urlEntity, mentre se il campo è un request param (tipo la budget) basta l'id